lib/cosmos/tools/test_runner/test_runner.rb

```diff
@@ -822,6 +822,7 @@ def show_select
                                              Qt::DialogButtonBox::Cancel)
         connect(button_box, SIGNAL('rejected()'), box, SLOT('reject()'))
         connect(button_box, SIGNAL('accepted()')) do
+          ScriptRunnerFrame.instance = @script_runner_frame
```

Comment:
You can't run instrumented code without ScriptRunnerFrame.instance being defined.  (And it is generally only supposed to be defined while a ScriptRunnerFrame is running).   In this case you are calling initialize (which is instrumented code) on a Test class, so ScriptRunnerFrame.instance needs to be defined.